ts/features/itwallet/common/saga/index.ts

```diff
@@ -27,6 +28,8 @@ export function* watchItwSaga(): SagaIterator {
     return;
   }
 
+  yield* call(checkHasNfcFeatureSaga);
+
```

Comment:
This should be placed before the wallet inconsistency saga (line 22), otherwise it may never get executed

Reply:
applied https://github.com/pagopa/io-app/pull/6985/commits/2023b9b0d5d95886ab6f4ccbe0aaee106f4d98b1